### PR TITLE
feat: add iOS escape hatch link + ui updates

### PR DESCRIPTION
**Pull request by shalanah**

### CODE REVIEW

**Comment by shalanah** [comment]
@Pinjasaur Take a look at the UI. Have any thoughts about it's incorporation?

**Comment by shalanah** [comment]
I haven't touched this repo for awhile so rolled some other updates into it.

File: README.md [hunk 1]
```diff
@@ -13,7 +13,17 @@ Test common in-app issues
 - ✅ Test in-app downloads (on-the-fly and static assets)
 - ✅ Test in-app escape links (Android + iOS)
 
-# Mobile App Tests
+## Running
+
+- `nvm use`
```
Comment (Pinjasaur) [inline README.md:18]:
```suggestion
- `nvm install`
```

nit: can just use `nvm install` to install-and-use or use-if-already-installed. I have a quickie blog post about this I need to publish sometime soon.

File: src/App.tsx [hunk 2]
```diff
@@ -12,6 +13,21 @@ const Div = styled.div`
 
 export const App = () => {
   if (typeof window === "undefined") return null; // make sure latest + greatest browser info
+
+  // polyfill crypto.randomUUID
```
Comment (Pinjasaur) [inline src/App.tsx:17]: didn't even consider a polyfill, I suppose browser support isn't 100% so nice addition https://caniuse.com/mdn-api_crypto_randomuuid

File: README.md (hunk 1 above)
Comment (shalanah) [reply README.md:18]: good to know! I'll put `nvm install` / `nvm use` for us old-timers 🤣

File: src/App.tsx (hunk 2 above)
Comment (shalanah) [reply src/App.tsx:17]: Yeah I wasn't even aware of crypto.randomUUID before this. Cool to learn about!

**Comment by Pinjasaur** [comment]
> @Pinjasaur Take a look at the UI. Have any thoughts about its incorporation?
> 
> [deploy-preview-3--inappdebugger.netlify.app](https://deploy-preview-3--inappdebugger.netlify.app/)

Just seeing this today—I took a peek and it looks great on desktop & mobile.

In your opinion do you think it's reliable enough to sneak in a quick update to the recent blog post? https://frontendmasters.com/blog/the-pitfalls-of-in-app-browsers/#escaping

It's not 100%, but by far the most reliable iOS strategy that I've came across. 🤷🏻

**Comment by shalanah** [comment]
@Pinjasaur 

I think it is reliable at least on the latest OS (iPad and MacOS too!). 
I wish I could test on iOS 16 and farther back.

It does feel hacky, and may get removed, but it's also awesome. 🎉
So I'm sure a disclaimer with it would be worth it unless we want to keep this hack to ourselves 🤣

Apple has a tendency of removing these hacks, but I'm sure a lot of shortcut people also depend on this one.

If they remove this, they should create an Apple-approved way to open in a default browser like the intent Android links.

I haven't incorporated this yet into my sites, but probably will since I still get several emails a day specifically from iPhone users having issue downloading.

**Pull request merged**